### PR TITLE
Add support for querying by name

### LINKED ISSUE #100: Allow names to be used in deps CLI
Once gh-99 is complete, we can update the CLI to support querying for names. This can be done by moving the organization and module parsing to the client-side and then sending the appropriate request. This should simplify the developer experience and provide a nice stepping stone to v1beta.

### PR DESCRIPTION
**Pull request by shinnlok**
Fixes #100

I wasn't exactly sure how to test.

### CODE REVIEW

File: deps/internal/client/http_modules.go [hunk 1]
```diff
@@ -39,11 +39,12 @@ func (m *httpModuleClient) List(ctx context.Context, in *tracker.ListRequest, op
 }
 
 func (m *httpModuleClient) ListSources(ctx context.Context, in *schema.Module, opts ...grpc.CallOption) (*tracker.ListSourcesResponse, error) {
```
Comment (mjpitz) [inline deps/internal/client/http_modules.go:41]: The HTTP clients are more or less a dead code path. I wouldn't bother trying to update them.

**Review by mjpitz — CHANGES_REQUESTED** [review]
Alright. So this is close, but for this to work properly, we need to ensure `organziation` and `module` are set properly on the request. I would write a helper function that accepts a `*Module` or `*DependencyRequest`. If the name is set, do the parsing that you're doing on top to extract the organization / module and set the values appropriately on the request.

The nice thing is that this helps simplify some of the checks a bit more.

Otherwise, this is

**Comment by shinnlok** [comment]
@mjpitz Ah that makes a lot more sense. Updated.

**Review by mjpitz — APPROVED** [review]
I think you have one bug, but otherwise, this changeset looks good to me.

File: deps/internal/cmds/get/common.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package get
 import (
 	"github.com/depscloud/api/v1alpha/schema"
 	"github.com/depscloud/api/v1alpha/tracker"
+	"strings"
```
[the remark this answers is not in the document]
Comment (shinnlok) [reply deps/internal/cmds/get/common.go]: 👍

File: deps/internal/cmds/get/common.go [hunk 3]
```diff
@@ -27,4 +29,51 @@ func addModuleFlags(cmd *cobra.Command, module *schema.Module) {
...
+	} else {
+		if strings.EqualFold(language, "node") {
+			name = strings.Replace(name, "@", "", 1)
+		}
+
+		split = strings.SplitN(name, "/", 2)
+	}
+
+	if len(split) == 1 {
+		return []string{"_", split[1]}
```
[the remark this answers is not in the document]
Comment (shinnlok) [reply deps/internal/cmds/get/common.go]: Oops. Thanks!

**Review by mjpitz — APPROVED** [review]

**Comment by mjpitz** [comment]
@shinnlok Awesome! Thank you so much for all the hard work. Would you mind taking a pass over the documentation and updating the queries to use the -n flag instead of organization and module?

**Comment by shinnlok** [comment]
@mjpitz done https://github.com/depscloud/deps.cloud/pull/74

I also noticed I missed one https://github.com/depscloud/depscloud/pull/118 while looking over the docs.

**Pull request merged**